StellaNowSDKDemo/src/Program.cs

```diff
@@ -61,6 +61,12 @@ static async Task Main(string[] args)
             stellaSdk.SendMessage(message);
             logger.LogInformation("New Message Queued");
 
+            logger.LogInformation(
```

Comment:
I do not understand how this works? this loop iterates up to `MessageCount` which is private and it's always 1. do you change this variable if you want to run test with eg 100 events? maybe it should be passed as `string[] args` argument?